### PR TITLE
Ethan - Add frontend components for the course-over-time search

### LINKED ISSUE #28: Add frontend components for the course-over-time search
Add Frontend Components for the form that asks the user for a quarter range, subject area and course number. You may be able to steal this, and it's tests and stories, from the old code base. Add these to storybook, along with fixtures (you may be able to use existing fixtures)

Requirements:
- [x] There is a search form which takes a start quarter, end quarter, subject area, and course number
- [ ] This form, upon submit, makes a call to the backend API defined in #27. This can be done in parallel to #27, as long as the API path (url mapping) is *consistent* between both
- [x] There are appropriate tests for this search form
- [x] There are stories & fixtures for the form & the search entity

### PR DESCRIPTION
**Pull request by ethanlee556**
Closes #28 
Added CourseOverTimeSearch.js which takes a start quarter, end quarter, subject, and course number as input to a form.
Link: https://ucsb-cs156-f22.github.io/f22-5pm-courses-docs-qa/storybook-qa/ethan-courseOTsearch/?path=/docs/components-basiccoursesearch-courseovertimesearch--default
image

### CODE REVIEW

**Review by liamjet — DISMISSED** [review]
LGTM

File: frontend/src/fixtures/courseOverTimeSearchFixtures.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      [
+        {
+          "startQuarter": "20084",
+          "endQuarter": "20214",
+          "subjectArea": "CMPSC ",
+          "courseNumber": "8  "
+        }
+    ]
+  }
+
```
Comment (pconrad) [inline frontend/src/fixtures/courseOverTimeSearchFixtures.js:27]: Are these used anywhere?  I couldn't find any place they were.

We typically create fixtures when we are mocking the JSON that is returned by a backend endpoint, but I don't see any backend endpoint that would be returning this; instead, these look like the parameters that we'd be passing to the backend.

If I'm missing something, let me know, but I'm wondering if we can delete this.

**Review by pconrad — CHANGES_REQUESTED** [review]
One small question/suggestion... otherwise, LGTM.

Once we resolve this one issue, I think this is mergable.

File: frontend/src/fixtures/courseOverTimeSearchFixtures.js (hunk 1 above)
Comment (ethanlee556) [reply frontend/src/fixtures/courseOverTimeSearchFixtures.js]: No they aren't. I deleted the file in my new commit.

**Review by liamjet — APPROVED** [review]
LGTM 2!

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**